spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/config/RabbitMessageChannelBinderConfiguration.java

```diff
@@ -21,8 +21,8 @@
 import org.springframework.amqp.support.postprocessor.DelegatingDecompressingPostProcessor;
 import org.springframework.amqp.support.postprocessor.GZipPostProcessor;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.autoconfigure.PropertyPlaceholderAutoConfiguration;
 import org.springframework.boot.autoconfigure.amqp.RabbitProperties;
+import org.springframework.boot.autoconfigure.context.PropertyPlaceholderAutoConfiguration;
```

Comment:
This is tricky, unfortunately. `PropertyPlaceholderAutoConfiguration` is deprecated, and yet we will want to use this codebase with 1.4 as well. We might ignore deprecation until a later stage or at least be able to alternatively pick import one class or another. I'd suggest we solve the deprecation as a separate PR.